src/rest/Facet/FacetRequest.ts

```diff
@@ -41,6 +45,22 @@ export interface IFacetRequestValue extends IRangeValue {
    * **Default:** `false`
    */
   preventAutoSelect?: boolean;
+  /**
+   * Whether to retrieve the children of this hierarchical facet value. Can only be used on leaf values.
+   * 
+   * **Default:** `false`
+   */
+  retrieveChildren?: boolean;
+  /**
+   * The maximum number of children to retrieve for this hierarchical facet value. Ignored if `retrieveChildren` is `false`.
+   */
+  retrieveCount?: number;
```

Comment:
Does this count apply to the maximum "horizontal" number of children? or does it refer to the "vertical"/recursive number of children?

Reply:
vertical! For those 2 I just copied the swagger documentation